configuration/README.md

```diff
@@ -28,6 +28,25 @@ Sources are identified via regexp according to package, type, and field names.
 }
 ```
 
+Sources may also be identified via field tags:
+```go
+type Example struct {
+	fieldName fieldType `levee:"source"` // this field will be considered a Source
+}
+```
+
+The tag `levee:"source"` is built-in. Additional tags may be identified via regexp:
+```json
+{
+	"FieldTags": [
+		{
+			"Key": "<key regexp>",
+			"Val": "<value regexp>"
```

Comment:
Do we want to do regexp here rather than explicit string matching?  I feel like tags are less volatile, and tags a user will want to match is likely to be a minimal set.  I could go either way on it, though.

Reply:
I feel the same.

I was essentially going for consistency with the rest of the config. Since everything else is a regexp, users may expect this to be as well. Regexp also allow users more flexibility if they need it.

I've added "RE" to the end of the JSON keys to make it more explicit that these are regexp.

WDYT?

Reply:
Either way.  We're going to end up revisiting it when config is redesigned in #89 et al in any case.  Something I'll keep in mind then.